rasterio/crs.py

```diff
@@ -54,10 +57,15 @@ def from_string(prjs):
     if '{' in prjs:
         # may be json, try to decode it
         try:
-            return json.loads(prjs, strict=False)
+            val = json.loads(prjs, strict=False)
         except ValueError:
             raise ValueError('crs appears to be JSON but is not valid')
```

Comment:
@geowurster I could go either way. `CRSError` seems more specific so I'm tempted to go with that.